flow/src/main/java/com/sequenceiq/flow/repository/FlowLogRepository.java

```diff
@@ -76,7 +76,7 @@ public interface FlowLogRepository extends CrudRepository<FlowLog, Long> {
     Set<String> findAllFlowIdsByChainIds(@Param("chainIds") Set<String> chainIds);
 
     @Query("SELECT fl FROM FlowLog fl WHERE fl.flowId IN (:flowIds) ORDER BY fl.created DESC")
-    List<FlowLog> findAllByFlowIdsCreatedDesc(@Param("flowIds") Set<String> flowIds);
+    Page<FlowLog> findAllByFlowIdsCreatedDesc(@Param("flowIds") Set<String> flowIds, Pageable pageable);
```

Comment:
tests are missing for this. We should use DataJpaTest which can also verify the paging.

Reply:
added test for this